controllers/memberoperatorconfig/memberoperatorconfig_controller.go

```diff
@@ -48,5 +51,58 @@ func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.
 	reqLogger := log.FromContext(ctx)
 	reqLogger.Info("Reconciling MemberOperatorConfig")
 
-	return reconcile.Result{}, loadLatest(r.Client)
+	crtConfig, err := loadLatest(r.Client)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
+	if err := r.handleAutoscalerDeploy(reqLogger, crtConfig, request.Namespace); err != nil {
+		return reconcile.Result{}, err
+	}
+
+	if err := r.handleUserPodsWebhookDeploy(reqLogger, crtConfig, request.Namespace); err != nil {
```

Comment:
Correct me if I'm wrong - to follow the best practices and also to properly manage the deployments, the controller should also watch the resources that are created/deleted here, right?

Reply:
Right. As discussed let's handle that in an upcoming PR for https://issues.redhat.com/browse/CRT-1194